lib/shopify_app/webhooks_manager.rb

```diff
@@ -37,6 +77,12 @@ def is_required_webhook?(webhook)
       required_webhooks.map{ |w| w[:address] }.include? webhook.address
     end
 
+    def is_valid_webhook?(webhook)
+      unless VALID_WEBHOOK_TOPICS.any? { |valid| valid == webhook }
+        Rails.logger.warn "#{webhook} isn't a valid topic. Valid topics include: #{VALID_WEBHOOK_TOPICS}"
+      end
+    end
```

Comment:
we could make this a concern that is included in both classes. Checkout ActiveSupport::Concern it should do the trick.